nixpkgs/default.nix

```diff
@@ -1 +1,6 @@
-import ((import <nixpkgs> {}).fetchFromGitHub (builtins.fromJSON (builtins.readFile ./github.json)))
+let
+  fetcher = { owner, repo, rev, sha256 }: builtins.fetchTarball {
```

Comment:
I would suggest using fetchFromGitHub as a backup when fetchTarball is not available. You can check this with ```(builtins ? fetchTarball)```.

Reply:
Thanks, updated!